drivers/include/lis2dh12.h

```diff
@@ -256,6 +323,85 @@ int lis2dh12_init(lis2dh12_t *dev, const lis2dh12_params_t *params);
  */
 int lis2dh12_read(const lis2dh12_t *dev, int16_t *data);
 
+/**
+ * @brief   Clear the LIS2DH12 memory
```

Comment:
What memory is there on the LIS2DH12 other than the FIFO?
Or will this just reset the configuration?